xml/System.Runtime.Intrinsics.X86/Sse41.xml

```diff
@@ -3145,7 +3145,7 @@ The native signature doesn't exist. This additional overload is provided for com
       </Parameters>
       <Docs>
         <param name="value">To be added.</param>
-        <summary>_MM_FROUND_CUR_DIRECTION; ROUNDPD xmm, xmm/m128, imm8(4)</summary>
+        <summary>_m128d _mm_round_pd (__m128d a, _MM_FROUND_CUR_DIRECTION); ROUNDPD xmm, xmm/m128, imm8(4)</summary>
```

Comment:
nit: Other docs have this split into two separate `<para>` tags:
```suggestion
        <summary>
           <para>_m128d _mm_round_pd (__m128d a, _MM_FROUND_CUR_DIRECTION)</para>
           <para>ROUNDPD xmm, xmm/m128, imm8(4)</para>
         </summary>
```

Reply:
Fixed by commit 908a236